.github/workflows/build.yml

```diff
@@ -1,5 +1,9 @@
 name: Build and Test
-on: push
+on:
+  push:
+    paths-ignore:
+      - docs/**
+      - .github/workflows/docs.yml
```

Comment:
this can be dangerous. due to the nature of how this works, anybody can push a small change to docs in an extra commit and that will remove any previously failed checks on the PR.
also if you make this workflow ever required for merging, you won't be able to merge docs changes.

i'd rather not use this at all.

Reply:
> also if you make this workflow ever required for merging

We won't be doing that just yet. Right now the only requirement is that a review be given on a PR. If someone decides to merge a PR when any checks are red or without checking failures, I would argue that's on whoever merged it. This is still a thin wrapper and until we have actual documentation being built + uploaded, I'd argue it's fine to keep this here so that #11 (which is a recreation of #8) can be worked on in a faster turn around time.

I'm willing to put more time into this later, but if you look at the repository you'll see there isn't a docs.yml or docs/** file

Reply:
`paths-ignore` actively hides failed checks on a PR, making it seemingly green when it is not.
it's actively harmful and i have a strong opinion we should not use it for branches that PRs are made with. doing this for `main` would be fine with me.

Reply:
I'll apply that for the PR then.